### PR TITLE
Remove warning about Quilt being unsupported with CurseForge modpacks

### PR DESCRIPTION
**Pull request by Trial97**
The  curseforge will support quilt as modpack :https://discord.com/channels/428228256236306434/525372338653626389/1133654748197818458
The export code is there presuming that they will use a similar approach as fabric(also the import is already done in that case)

### CODE REVIEW

**Comment by getchoo** [comment]
this is probably right, but i think it's best to wait until quilt modpacks are uploaded to cf (if they aren't already) and make sure :p

File: launcher/ui/MainWindow.cpp [hunk 1]
```diff
@@ -1342,16 +1342,10 @@ void MainWindow::on_actionExportInstanceFlamePack_triggered()
...
-            if (instance->getPackProfile()->getComponent("org.quiltmc.quilt-loader")) {
-                errorMsg = tr("Quilt is currently not supported by CurseForge modpacks.");
-            } else if (auto cmp = instance->getPackProfile()->getComponent("net.minecraft");
-                       cmp && cmp->getVersionFile() && cmp->getVersionFile()->type == "snapshot") {
-                errorMsg = tr("Snapshots are currently not supported by CurseForge modpacks.");
-            }
-            if (!errorMsg.isEmpty()) {
+            if (auto cmp = instance->getPackProfile()->getComponent("net.minecraft");
+                cmp && cmp->getVersionFile() && cmp->getVersionFile()->type == "snapshot") {
                 QMessageBox msgBox;
```
Comment (TheKodeToad) [inline launcher/ui/MainWindow.cpp]:
```suggestion
                QMessageBox msgBox(this);
```
might as well do this now?

**Comment by TheKodeToad** [comment]
> this is probably right, but i think it's best to wait until quilt modpacks are uploaded to cf (if they aren't already) and make sure :p

would be so easy to check if my installation of cf had access to quilt :/

**Review by TheKodeToad — APPROVED** [review]
I very much doubt this won't work. I guess there's still a chance...

**Review by Scrumplex — APPROVED** [review]

**Comment by RaptaG** [comment]
CF now supports Quilt, this can be merged

**Comment by Trial97** [comment]
> CF now supports Quilt, this can be merged

I'm not seeing any quilt modpack on curseforge. Can you point me to one in order to test this?

**Comment by TheKodeToad** [comment]
image
(yes I did have to use Windows 7 and Notepad)

**Comment by DioEgizio** [comment]
> image (yes I did have to use Windows 7 and Notepad)

shouldn't we also allow to actually install them? or another commit i've missed already does that?

**Comment by TheKodeToad** [comment]
\*Apparently* it's already implemented

**Comment by Trial97** [comment]
The quilt was supported in the initial curseforge export implementation(added the warning at your request). And for import I reworked the version checking in the use metadata for recommended version PR.

**Pull request merged**